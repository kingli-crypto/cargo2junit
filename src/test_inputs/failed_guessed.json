{ "type": "suite", "event": "started", "test_count": 2 }
{ "type": "test", "event": "started", "name": "tests::fails" }
{ "type": "test", "event": "started", "name": "tests::success" }
{ "type": "test", "name": "tests::fails", "event": "failed", "stdout": "Some log line\nSome log line\nSome more log line\nError: system failure" }
{ "type": "test", "name": "tests::success", "event": "ok" }
{ "type": "suite", "event": "failed", "passed": 1, "failed": 1, "allowed_fail": 0, "ignored": 0, "measured": 0, "filtered_out": 0 }
