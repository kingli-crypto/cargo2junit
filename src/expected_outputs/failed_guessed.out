<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite id="0" name="cargo test #0" package="testsuite/cargo test #0" tests="2" errors="0" failures="1" hostname="localhost" timestamp="2023-08-18T11:58:25.815589Z" time="0">
    <testcase name="fails" time="0" classname="tests">
      <failure type="cargo test" message="Error: system failure"><![CDATA[Some log line
Some log line
Some more log line
Error: system failure]]></failure>
      <system-out><![CDATA[Some log line
Some log line
Some more log line
Error: system failure]]></system-out>
    </testcase>
    <testcase name="success" time="0" classname="tests" />
  </testsuite>
</testsuites>
